includes/abstracts/abstract.llms.post.model.php

```diff
@@ -330,15 +330,16 @@ private function create( $title = '' ) {
 
 	/**
 	 * Clones the Post if the post is cloneable
-	 * @return   mixed         WP_Error or WP Post ID of the clone (new) post
+	 * @return   WP_Error|int|false WP_Error, WP Post ID of the clone (new) post, or false if post is not cloneable
 	 * @since    3.3.0
-	 * @version  3.19.2
+	 * @since    [version] Returns false if post is not cloneable.
+	 * @version  [version]
 	 */
 	public function clone_post() {
 
-		// if post type doesnt support cloning don't proceed
+		// if post type doesn't support cloning, don't proceed
 		if ( ! $this->is_cloneable() ) {
-			return;
+			return false;
```

Comment:
Is there some reason eluding me that we need to explicitly define `false` instead of just changing the documentation to reflect the way the code is actually working currently?

Reply:
I consider it bad programming practice for a function that normally returns a value to also possibly return void. I read about this years ago, but can't seem to find any web pages about this topic right now. PHPStorm warns that a missing return value (in functions that can also return a value) may be a programmer mistake.

After more thought, I think I'll just change it to return null to keep this method's behavior the same and make PHPStorm happy. If someone wants to test for a uncloneable condition in the future, we could change it to return false, return WP_Error or throw an exception.